applications/daos.md

```diff
@@ -0,0 +1,135 @@
+|1.       | create-dao pallet | Developers can quickly create a behavior template that works with daos, and users can create DAOs based on behavior templates in this module.                               |
+|2.        | agency pallet     | A power agent in DAO can solve some things in DAO more quickly. The module can set the difficulty level of transactions that can be executed by power agent, namely Origin. Has its own voting system|
+|3.     | doas pallet       | Through this module, the agency can obtain a dao-account, which is the identity of an ordinary user to perform external transactions                                        |
+|4.     | square pallet     | The highest authority in dao, can execute all transactions in dao. And the execution difficulty of each transaction can be set in this modlue, namely vote weight. Has its own voting system         |
+|5.     | sudo pallet       | Giving someone root privileges can execute all executable methods in dao. It should be remove when the DAO is necessary for decentralization.                               |
+### Milestone 2 - Emergency, UI design and JS code implementation examples
+* **Estimated duration:** 1 month
+* **FTE:** 3
+* **Costs:** 4000 USD
+
+|Number   | Deliverable                         | Specification                                                                                                                    |
+|-----:   |-------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+|0a. | Documentation                       | Provide documentation for using daos                                                                                             |
+|0b. | Testing                             | Complete all module tests                                                                                                        |
+|0c. | Comment                             | Comment for each module                                                                                                          |
+|0d. | weight                              | Implement default weight for each module                                                                                         |
+|1.       | emergency pallet                    | When dao cannot run normally for some reason, such as wrong parameter settings, it can be adjusted through democracy and council |
+|2.       | Add more `set_xxx` in these pallets | More parameters can be set directly and governance is more flexible                                                              |
+|3.       | UI design                           | Providing a UI framework                                                                                                         |
```

Comment:
Can you expand on this? What technologies/frameworks are you planning to use? Is this a finished UI, or rather a library for others to build one?

Reply:
> Can you expand on this? What technologies/frameworks are you planning to use? Is this a finished UI, or rather a library for others to build one?

This part will be out soon, we are still working on it at the moment. The kico team is already developing a front-end implementation based on vc dao. At that time, our two teams will conduct a comparison and then provide the public part to the community. Currently, they are using Flutter

Reply:
Does this UI include template creation and selection? I like the idea of templates, but I'm wondering what it will look like from an end-user perspective. Governance is difficult to understand as it is, so I would be interested to know how you are going to present these templates to the user when they have to choose a template. Have you thought about this?

Reply:
@JimYam

> Does this UI include template creation and selection?

Do you have an answer for that yet?

> This part will be out soon, we are still working on it at the moment.

Are you planning to share more details as part of the application anytime soon?